pawno/include/modules/src/gunmenu.inc

```diff
@@ -198,47 +198,73 @@ ShowGunmenuDialog(playerid)
+					format(str, sizeof str, "%s\n{696969}%d\t{696969}%s\t{696969}%d\t{696969}%d / %d", str, i, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][GunAmmo], GunmenuData[i][TimesSelected][team], GunmenuData[i][GunLimit]);
 				}
 			}
-			ShowPlayerDialog(playerid, DIALOG_GUNMENU, DIALOG_STYLE_TABLIST_HEADERS, sprintf("%sGunmenu", TextColor[team]), str, sprintf("%sPick", TextColor[team]), sprintf("%sReset", TextColor[team]));
+			
+			ShowPlayerDialog(playerid, DIALOG_GUNMENU, DIALOG_STYLE_TABLIST_HEADERS, "{FFFFFF}Select your weapons:", str, "Pick", "Reset");
 		}
 		case ARENA:
 		{
 			new team = Player[playerid][Team];
-			new str[550];
-			format(str, sizeof str, "%sWeapon\t%sTimes Picked", TextColor[team], TextColor[team]);
+			new str[800];
+			
+			format(str, sizeof str, "{FFFFFF}ID\t{FFFFFF}Weapon\t{FFFFFF}Times Picked");
+			
 			for(new i = 0; i < MAX_GUNMENU_GUNS; i ++)
 			{
 				// Do some coloring for items
 				if(GunmenuData[i][HasGun][playerid]) // already got this weapon
 				{
-					format(str, sizeof str, "%s\n{DB5858}%s\t%d", str, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
+					format(str, sizeof str, "%s\n{FFFFFF}%d\t{FFFFFF}%s\t{FFFFFF}%d", str, i, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
 				}
 				else if(GunmenuData[i][GunLimit] <= 0) // not available
 				{
-					format(str, sizeof str, "%s\n{696969}%s\t%d", str, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
+					format(str, sizeof str, "%s\n{696969}%d\t{696969}%s\t{696969}%d", str, i, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
 				}
 				else // available
 				{
-					format(str, sizeof str, "%s\n{50EB50}%s\t%d", str, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
+				    if(i%2 == 0)
+					{
+					    if(team == ATTACKER) format(str, sizeof str, "%s\n{FFFFFF}%d\t{FF2222}%s\t{FF2222}%d", str, i, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
+					    else format(str, sizeof str, "%s\n{FFFFFF}%d\t{5555FF}%s\t{5555FF}%d", str, i, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
+					}
+					else
+					{
+					    if(team == ATTACKER) format(str, sizeof str, "%s\n{FFFFFF}%d\t{FF4444}%s\t{FF4444}%d", str, i, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
+					    else format(str, sizeof str, "%s\n{FFFFFF}%d\t{7777FF}%s\t{7777FF}%d", str, i, WeaponNames[GunmenuData[i][GunID]], GunmenuData[i][TimesSelected][team]);
+					}
 				}
 			}
-			ShowPlayerDialog(playerid, DIALOG_GUNMENU, DIALOG_STYLE_TABLIST_HEADERS, sprintf("%sGunmenu", TextColor[team]), str, sprintf("%sPick", TextColor[team]), sprintf("%sReset", TextColor[team]));
+			
+			ShowPlayerDialog(playerid, DIALOG_GUNMENU, DIALOG_STYLE_TABLIST_HEADERS, "{FFFFFF}Select your weapons:", str, "Pick", "Reset");
 		}
 	}
 }
```

Comment:
Nah I don't think this is necessary. Not much notable, I mean, so please remove it.